spec/models/test_track/session_spec.rb

```diff
@@ -369,7 +366,7 @@ def current_clown; end
 
               expect(Thread).to have_received(:new)
               expect(TestTrack::UnsyncedAssignmentsNotifier).to have_received(:new) do |args|
-                expect(args[:mixpanel_distinct_id]).to eq("fake_distinct_id")
+                expect(args.keys).to contain_exactly :visitor_id, :assignments
```

Comment:
I think we can remove this line, we don't need to overly specify what args it expects.